harrison-GitTest4.md

```diff
@@ -0,0 +1,5 @@
+# Harrison Lilley- GitTest #4
+
+Hello everyone,
+
+For this assignment, I decided to look at the Dracula and the Blues Analysis Project. The methods and technologies that were used but were not limited to in the research were utilizing the "Find and Replace in Files" in Oxygen, using online tutorials for HTML, and the usage of XSLT. The features and visualizations about these projects that were interesting to me were the usage of visual diagrams, or in some way flow charts (I am not actually 100% sure what they are called). These visualizations show things such as the Network of Relationships between Songs and the decade that they were written in, and to show a visual "network" of everything that was related to Dracula. Personally, seeing a picture makes it a lot easier for me to learn a concept instead of just a boring lecture.
```

Comment:
Hey Harrison! I believe you are referring to the `Network Graphs`. They display the relationship between different words and characters.